### DISCUSSION TITLE
URL Scheme can pass additional arguments to the refreshed plugin

### DISCUSSION THREAD

**Discussion by futurist** [swiftbar/SwiftBar#368] [Ideas]
Think below command:

```sh
open -g 'swiftbar://refreshplugin?name=myplugin&abc=1&def=2'
```

`myplugin` should accept `abc=1&def=2` as param or environment variable.

That way other programs can communicate with the plugin in system wide(like from a webpage or webview controller)

**Comment by melonamin** [comment]
Added to backlog #369
